suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx

```diff
@@ -43,24 +45,19 @@ export const AccountDetailScreenHeader = ({
 
     return (
         <ScreenHeader
-            leftIcon={
-                <IconButton
-                    colorScheme="tertiaryElevation0"
-                    size="medium"
-                    iconName="chevronLeft"
-                    onPress={() => navigation.goBack()}
-                />
-            }
+            hasGoBackIcon
             rightIcon={
-                <IconButton
-                    colorScheme="tertiaryElevation0"
-                    size="medium"
-                    iconName="settings"
-                    onPress={handleSettingsNavigation}
-                />
+                !tokenName && (
```

Comment:
please create some extra variable for this like `const isToken = !!tokenName` is easier to understand then

Reply:
fixed in https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25